scripts/README.md

```diff
@@ -0,0 +1,65 @@
+# Test Generator
+
+This script will help you port over an exercise from the [Problem Specifications Repo](https://github.com/exercism/problem-specifications).
+
+The script is run using Haxe's built in interpreter with the `--run` command.  
+
+:warning: **For this generator to work the exercise being ported must have a canonical-data.json.**
+
+The generator takes 1 argument:
+
+```
+exercise: The exercise name exactly as it is found in the Problem Spec Repo.
+```
+
+An example to generate the acronym exercise:  
+
+```
+./scripts/gen-test.sh acronym
+```
+
+You can also enter the `scripts` directly and use either of the following directly:
+
+```
+haxe testgen.hxml acronym
+```
+
+```
+haxe --run TestGen.hx acronym 
+```
+
+If you are on Windows you will need to compile it first to a Sys target due an SSL issue.
+
+Neko compile:
+```
+haxe.exe --neko testgen.n --main TestGen.hx
+```
+
+Neko run:
+```
+neko.exe testgen.n anagram
+```
+
+
+Workflow:
```

Comment:
The first step should be to add the exercise to the track's config.json. That allows us to run configlet sync inside this script to set things up. Then they should proof the tests before updating Example.hx. They should update the exercise's config.json to add themselves as an author, and then make the PR.